h2/src/main/org/h2/engine/Database.java

```diff
@@ -315,6 +316,8 @@ public Database(ConnectionInfo ci, String cipher) {
                 TraceSystem.DEFAULT_TRACE_LEVEL_SYSTEM_OUT);
         this.cacheType = StringUtils.toUpperEnglish(
                 ci.removeProperty("CACHE_TYPE", Constants.CACHE_TYPE_DEFAULT));
+        this.ignoreCatalogs = ci.getProperty("IGNORE_CATALOGS",
```

Comment:
You should use `getProperty(String, boolean)` here.

Reply:
Thinking too complicated